web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -72,7 +73,7 @@
         WebReflectionUtils.initialize();
     }
     // region Members
-    private final static String FILTER_NAME = "ApplicationInsightsWebFilter";
+    final static String FILTER_NAME = "ApplicationInsightsWebFilter";
```

Comment:
Was this made non private for testing purpose? Ok It's fine I think. Class is final so no one will extend it. By the way, cam protected work here?

Reply:
Yes, it's `@VisibleForTesting`...I should've added that